go/vt/vttablet/tabletmanager/rpc_vreplication.go

```diff
@@ -26,7 +26,7 @@ import (
 
 // VReplicationExec executes a vreplication command.
 func (tm *TabletManager) VReplicationExec(ctx context.Context, query string) (*querypb.QueryResult, error) {
-	qr, err := tm.VREngine.Exec(query)
+	qr, err := tm.VREngine.Exec(query, true)
```

Comment:
To make this change less intrusive to all existent callers, I would recommend doing the following:

* Add a method called `tm.VRengine.ExecWithDba(query)`.  We this method here.
* All the other call sites, continue to use ` tm.VRengine.Exec`.
* Inside VREngine, you have private method `exec`,  that accepts the boolean. In that way the implementation of the new two methods is something like this:
```
func (vre *Engine) ExecWithDba(query string) (*sqltypes.Result, error) {
exec(query, true)
}
func (vre *Engine) Exec(query string) (*sqltypes.Result, error) {
exec(query, false)
}
func (vre *Engine) exec(query string, runAsAdmin bool) (*sqltypes.Result, error) {
existent...
}
```

One nit, we use a convention we having functions that accept bool, that when calling it we add a comment to indicate what that boolean value represents:

```
	qr, err := tm.VREngine.Exec(query, true /* runAsAdmin */  )
```